pycon/finaid/utils.py

```diff
@@ -59,10 +59,9 @@ def email_address():
     applications should send emails to with questions, etc.
 
     Default is ``pycon-aid@python.org``. Override by setting
-    FINANCIAL_AID['email'].
+    FINANCIAL_AID_EMAIL.
     """
-    return getattr(settings, "FINANCIAL_AID", {})\
-        .get('email', DEFAULT_EMAIL_ADDRESS)
+    return getattr(settings, "FINANCIAL_AID_EMAIL", DEFAULT_EMAIL_ADDRESS)
```

Comment:
What do you think about removing `DEFAULT_EMAIL_ADDRESS`, since this setting is defined for all settings files?